_includes/slider-init.html

```diff
@@ -0,0 +1,6 @@
+<script
+  defer
+  type="text/javascript"
+  src="https://cdnjs.cloudflare.com/ajax/libs/tiny-slider/2.9.2/min/tiny-slider.js"
+></script>
+<script defer type="text/javascript" src="/assets/js/slider.js"></script>
```

Comment:
Are we using the same JS libraries / code for all the sliders on HabitatMap?

Reply:
yes, same library